src/python/grpcio/grpc/_server.py

```diff
@@ -769,10 +869,28 @@ class _ServerStage(enum.Enum):
 
 
 class _ServerState(object):
+    lock: threading.RLock
+    completion_queue: cygrpc.CompletionQueue
+    server: cygrpc.Server
+    generic_handlers: Optional[List[grpc.GenericRpcHandler]]
```

Comment:
I don't think this needs to be `Optional`.

Reply:
Changed to `List[grpc.GenericRpcHandler]`.